storybook/src/stories/Global/TopBar.stories.tsx

```diff
@@ -132,11 +157,14 @@ export const _TopBar = () => {
         logoutLink,
         searchPlaceholder,
         hasLanguage,
+        hasUserDetails,
         hasCurrentUser,
         notificationsHistory,
         currentUserText,
-        logoutText
+        version,
+        logoutText,
       }}
+        userDetails={userConfig}
```

Comment:
Please change the variable names once you done with all other comments given above.

Reply:
Done